target/linux/bcm63xx/patches-5.10/532-MIPS-BCM63XX-add-inventel-Livebox-support.patch

```diff
@@ -134,8 +134,9 @@ Subject: [PATCH] MIPS: BCM63XX: add inventel Livebox support
 +{
 +	u8 *p;
 +	int count;
++	u8 m[ETH_ALEN] = { 0xBE, 0xBF, 0xF3, 0x77 };
 +
-+	memcpy(mac, (u8 *)0xBEBFF377, ETH_ALEN);
++	mac = m;
```

Comment:
This looks wrong. You assign pointer to a local array and modify that array later. The whole point of `livebox_get_mac_address()` is to modify array pointed by a caller.

Reply:
A previous version of this patchset made mac a u8[ETH_ALEN] array, which is more correct anyway. But since it has nothing to do with the failure, I removed it.